src/Button.js

```diff
@@ -7,6 +7,8 @@ class Button extends React.Component {
         accent: PropTypes.bool,
         className: PropTypes.string,
         colored: PropTypes.bool,
+        component: PropTypes.any,
```

Comment:
BTW, take a look at `TransitionGroup` component which is included in react in 0.13.